pjsip/src/pjsua2/endpoint.cpp

```diff
@@ -583,6 +663,12 @@ void Endpoint::utilLogWrite(LogEntry &entry)
     }
 }
 
+pj_status_t Endpoint::onCredAuth(OnCredAuthParam &prm)
+{
+    PJ_UNUSED_ARG(prm);
+    return PJ_FALSE;
```

Comment:
Return type is `pj_status_t`, better returning `PJ_ENOTSUP` instead of `PJ_FALSE`. Also note that: `PJ_FALSE == PJ_SUCCESS == 0 `.